<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Name</class>
 <widget class="QDialog" name="Name">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Enter your Name here :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameinput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="askName">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Go To Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
